src/Sparrow/LowMemory/LowMemoryNotification.cs

```diff
@@ -130,6 +131,7 @@ public static void Initialize(Size lowMemoryThreshold, float commitChargeThresho
         private readonly ManualResetEvent _simulatedLowMemory = new ManualResetEvent(false);
         private readonly ManualResetEvent _shutdownRequested = new ManualResetEvent(false);
         private readonly List<WeakReference<ILowMemoryHandler>> _inactiveHandlers = new List<WeakReference<ILowMemoryHandler>>(128);
+    private static readonly SmapsReader SmapsReader = new SmapsReader(new []{new byte[SmapsReader.BufferSize], new byte[SmapsReader.BufferSize]});
```

Comment:
Remove this. This is _going_ to cause us threading issues.
Instead, create this as a _local_ variable in the top function

Reply:
how this is better then creating the buffer inside SmapsReader ?